docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.7'
+
+services:
+  app:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    entrypoint:
+      - "npm"
+    command: ["start"]
```

Comment:
Use the CMD sytnax in the dockerfile to run this when the container starts
```suggestion
```

Reply:
I think we want this when running docker-compose for local development, but not when running the Jenkins builds.

Reply:
Yeah - i think that should be our default of how the image works and we override the jenkins behaviour via the `inside()` to run sh cmds and deploy via npm. 

This implies the docker image should be for us to run our dev server and setup a reproducible build env rather than we modify the behaviour for our dev env. We can still use that to build env in the docker image to deploy the code via jenkins.